From 40dabfae664f0dcc68bee588257197f28bac4fde Mon Sep 17 00:00:00 2001
From: Josef Schlehofer <pepe.schlehofer@gmail.com>
Date: Wed, 21 Jul 2021 22:39:58 +0200
Subject: [PATCH 1/4] foris-ws: update to version 1.6.2

In this version, there was moved ubus socket based on OpenWrt change
---
 web/foris-controller/foris-ws/Makefile | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/web/foris-controller/foris-ws/Makefile b/web/foris-controller/foris-ws/Makefile
index 0523cf08b..89359959b 100644
--- a/web/foris-controller/foris-ws/Makefile
+++ b/web/foris-controller/foris-ws/Makefile
@@ -8,12 +8,12 @@
 include $(TOPDIR)/rules.mk
 
 PKG_NAME:=foris-ws
-PKG_VERSION:=1.6.1
-PKG_RELEASE:=5
+PKG_VERSION:=1.6.2
+PKG_RELEASE:=1
 
 PKG_SOURCE_PROTO:=git
 PKG_SOURCE_URL:=https://gitlab.nic.cz/turris/foris-controller/foris-ws.git
-PKG_MIRROR_HASH:=ea9d5c059f0198d34cdde1eda1e4bd4ffd2d0e43e2676c732f65bc0c5ecd2f3f
+PKG_MIRROR_HASH:=9ed6c5ce90b0a11bacb9cd57a26a9fcf4ffa768bb4a58f5f221fb3627ea5e1e2
 PKG_SOURCE_VERSION:=v$(PKG_VERSION)
 
 PKG_MAINTAINER:=CZ.NIC <packaging@turris.cz>
-- 
2.30.2

